site/docs/v1/tech/apis/system.adoc

```diff
@@ -354,4 +356,59 @@ The specific contents of the JSON body are not documented here. If you choose to
 |===
 
 
+[NOTE]
+====
+This API has been available since 1.26.0
+====
+
+== Retrieve System Metrics Using Prometheus
+
+This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to setup Prometheus with the FusionAuth metrics endpoint.
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
+=== Authentication
```

Comment:
I guess this is okay to explain it here, since it is the only API call which supports basic auth. If we do more basic auth endpoints, we probably want to have a unique icon for basic auth.

Reply:
This can be authenticated using an API key or API key with Basic auth, we should list both.